db/migrate/20160927230136_enable_fk_message_digests_nodes.rb

```diff
@@ -0,0 +1,21 @@
+class EnableFkMessageDigestsNodes < ActiveRecord::Migration
+
+  class Bag < ActiveRecord::Base
+  end
+
+  class MessageDigest < ActiveRecord::Base
+    belongs_to :bag
+  end
+
+  def up
```

Comment:
I think I'm the only guinea pig trying to use migrations on an existing db on a DPN-dev node.  So we might be OK on this one.